src/styles/styles.ts

```diff
@@ -2388,7 +2444,15 @@ const styles = (theme: ThemeDefault) =>
             borderRadius: 88,
         },
 
-        rootNavigatorContainerStyles: (isSmallScreenWidth: boolean) => ({marginLeft: isSmallScreenWidth ? 0 : variables.sideBarWidth, flex: 1} satisfies ViewStyle),
+        rootNavigatorContainerStyles: (isSmallScreenWidth, isHomeOdSelected) => {
```

Comment:
Since this was changed elsewhere, let's keep the name consistent. `isOldDotHomeSelected`

```suggestion
        rootNavigatorContainerStyles: (isSmallScreenWidth, isOldDotHomeSelected) => {
```